kube/minkube/deployments/redis-deployment.yaml

```
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2023 KernelCI Authors.
+# Author: Sanskar Bhushan <sbdtu5498@gmail.com>
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kernelci-api-redis
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kernelci-api-redis
+  template:
+    metadata:
+      labels:
+        app: kernelci-api-redis
+    spec:
+      containers:
+        - name: kernelci-api-redis
+          image: redis:6.2
+          volumeMounts:
+            - name: redis-data
+              mountPath: /data
+          # Resource limits needs to be discussed.
+          resources:
+      volumes:
+        - name: redis-data
+          hostPath:
+            path: /home/docker/kernelci-api/redis/data
```

Mount volume `data` directory for `redis` is located inside `kernelci-api/docker/`. 
```suggestion
            path: /home/docker/kernelci-api/docker/redis/data
```

---

I just checked, there is no docker/redis folder. And the redis/data folder is present in the node. Please check.

---

@sbdtu5498 Yes, that is true that `docker/redis` is not present in the git source tree. But please check `docker-compose.yaml`.
```
volumes:
      - './docker/redis/data:/data'
```
So when `redis` service is up, it will create a directory `docker/redis/data` just like `ssh` and `storage` services.

---

I have added a commit to fix it.